lib/fields.c

```diff
@@ -43,12 +43,15 @@ int valid_field (const char *field, const char *illegal)
 			break;
 		}
 	}
-
+	
```

Comment:
Remove the indentation for the empty line.

Reply:
Removed indentation.